iocage_lib/ioc_fetch.py

```diff
@@ -129,13 +129,32 @@ def __fetch_eol_check__():
 
         return eol_releases
 
-    def __fetch_validate_release__(self, releases):
+    def __fetch_validate_release__(self, releases, eol=None):
         """
         Checks if the user supplied an index number and returns the
         RELEASE. If they gave us a full RELEASE, we make sure that exists in
         the list at all.
         """
 
+        host_release = iocage_lib.ioc_common.get_host_release()
+        for r in releases:
+            message = f'[{releases.index(r)}] {r}'
+            if r in eol or []:
```

Comment:
If `eol` is None this will cause a TypeError
```suggestion
            if eol is not None and (r in eol or []):
```

Reply:
Good catch 😉